R/stats-anova-tidiers.R

```diff
@@ -103,7 +103,36 @@ tidy.aov <- function(x, ...) {
 # it's not clear to me if glancing at an aov object should
 # return the same glance info as glance at the corresponding
 # regression. tentatively i'm calling this an error.
-glance.aov <- glance.default
+# 
+#' @templateVar class lm
+#' @template title_desc_glance
+#'
+#' @inherit tidy.aov params examples
+#'
```

Comment:
Can you add a comment in here about how the return has changed from 0.5.1 to 0.7.0? Should also update `NEWS`.